src/components/thumb-carousel/thumb-carousel.scss

```diff
@@ -52,13 +52,39 @@ $carousel-background: lighten($developer-blue, 40%);
       }
     }
 
+    .wrapper {
+      display: flex;
+      justify-content: center;
+    }
+
     .caption {
       font-weight: 300;
+      margin: 0;
+      padding: 0 10px;
       text-align: center;
     }
 
+    .quote {
+      width: 20px;
+      height: 20px;
+      flex-shrink: 0;
+      // Base 64 encoded quote glyph
+      background: url(data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4NCjwhLS0gR2VuZXJhdG9yOiBBZG9iZSBJbGx1c3RyYXRvciAxOS4xLjAsIFNWRyBFeHBvcnQgUGx1Zy1JbiAuIFNWRyBWZXJzaW9uOiA2LjAwIEJ1aWxkIDApICAtLT4NCjxzdmcgdmVyc2lvbj0iMS4xIiBpZD0iTGF5ZXJfMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgeD0iMHB4IiB5PSIwcHgiDQoJIHZpZXdCb3g9IjAgMCAxMDAgMjYiIHN0eWxlPSJlbmFibGUtYmFja2dyb3VuZDpuZXcgMCAwIDEwMCAyNjsiIHhtbDpzcGFjZT0icHJlc2VydmUiPg0KPHN0eWxlIHR5cGU9InRleHQvY3NzIj4NCgkuc3Qwe2ZpbGw6IzQyN0VBODt9DQoJLnN0MXtmaWxsOm5vbmU7fQ0KPC9zdHlsZT4NCjxnPg0KCTxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik0wLDE0LjRjMC0xLjIsMC4zLTIuMiwxLjItNC4yTDYsMGg0LjNMNy41LDEwLjhjMS4yLDAuOSwxLjksMi4yLDEuOSwzLjZjMCwyLjUtMi4yLDQuNS00LjcsNC41DQoJCUMyLjEsMTguOSwwLDE2LjksMCwxNC40eiBNMTIuOCwxNC40YzAtMS4yLDAuMy0yLjIsMS4yLTQuMkwxOC45LDBoNC4zbC0yLjksMTAuOGMxLjIsMC45LDEuOSwyLjIsMS45LDMuNmMwLDIuNS0yLjIsNC41LTQuNyw0LjUNCgkJQzE0LjksMTguOSwxMi44LDE2LjksMTIuOCwxNC40eiIvPg0KPC9nPg0KPHJlY3QgaWQ9Il94M0NfU2xpY2VfeDNFXyIgY2xhc3M9InN0MSIgd2lkdGg9IjI0IiBoZWlnaHQ9IjE5Ii8+DQo8L3N2Zz4NCg==);
```

Comment:
This is a prety cruftful SVG (thanks, illustrator!):
```
<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 19.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 100 26" style="enable-background:new 0 0 100 26;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#427EA8;}
	.st1{fill:none;}
</style>
<g>
	<path class="st0" d="M0,14.4c0-1.2,0.3-2.2,1.2-4.2L6,0h4.3L7.5,10.8c1.2,0.9,1.9,2.2,1.9,3.6c0,2.5-2.2,4.5-4.7,4.5
		C2.1,18.9,0,16.9,0,14.4z M12.8,14.4c0-1.2,0.3-2.2,1.2-4.2L18.9,0h4.3l-2.9,10.8c1.2,0.9,1.9,2.2,1.9,3.6c0,2.5-2.2,4.5-4.7,4.5
		C14.9,18.9,12.8,16.9,12.8,14.4z"/>
</g>
<rect id="_x3C_Slice_x3E_" class="st1" width="24" height="19"/>
</svg>
```
We can tighten it a lot, by using:
```
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 26">
<path fill="#427EA8" d="M0,14.4c0-1.2,0.3-2.2,1.2-4.2L6,0h4.3L7.5,10.8c1.2,0.9,1.9,2.2,1.9,3.6c0,2.5-2.2,4.5-4.7,4.5 C2.1,18.9,0,16.9,0,14.4z M12.8,14.4c0-1.2,0.3-2.2,1.2-4.2L18.9,0h4.3l-2.9,10.8c1.2,0.9,1.9,2.2,1.9,3.6c0,2.5-2.2,4.5-4.7,4.5 C14.9,18.9,12.8,16.9,12.8,14.4z"/>
<rect fill="none" width="24" height="19"/>
</svg>
```
And because it's text data, we don't even need to base64 encode it, the data-uri will already work:
```
background: url('data:image/svg+xml,<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 26"><path fill="#427EA8" d="M0,14.4c0-1.2,0.3-2.2,1.2-4.2L6,0h4.3L7.5,10.8c1.2,0.9,1.9,2.2,1.9,3.6c0,2.5-2.2,4.5-4.7,4.5 C2.1,18.9,0,16.9,0,14.4z M12.8,14.4c0-1.2,0.3-2.2,1.2-4.2L18.9,0h4.3l-2.9,10.8c1.2,0.9,1.9,2.2,1.9,3.6c0,2.5-2.2,4.5-4.7,4.5 C14.9,18.9,12.8,16.9,12.8,14.4z"/><rect fill="none" width="24" height="19"/></svg>');
```
saves space, and in a bizar data-uri twist, is more readable/debuggable too =)